### DISCUSSION TITLE
Push Failed-Telebot

### DISCUSSION THREAD

**Discussion by dinaraulliyah** [teamalgoritma/community#384] [Q&A] [answered]
first error: 
![gambar](https://user-images.githubusercontent.com/86821236/193458205-bfc20ee2-ef8c-4729-bb58-7cba21493e2b.png)
second error:
![gambar](https://user-images.githubusercontent.com/86821236/193458158-a9d440e0-4d6f-4249-93d9-29b5112f0d8b.png)

**Comment by yosiaazarya** [comment, marked as answer]
Halo selamat pagi Ibu, mohon dipastikan terlebih dahulu semua file yang sudah terupload dengan baik ke github ya. Terima kasih.
